src/naive_impl/mod.rs

```diff
@@ -3,6 +3,9 @@ pub mod canonical_kmer_iterator;
 mod kmer;
 
 pub mod hash;
+
+// Simple-sds does not compile on windows, so we make seq_vector an optional feature
+#[cfg(feature = "seq-vector")]
```

Comment:
@theJasonFan This looks nice and clean (love how Rust handles this).  My question is where do we _use_ `seq_vector`?  We probably have to conditionally include those methods as well, right?

Reply:
The library builds and tests pass even when `--features seq_vector` is not passed to `cargo test` so I think everything is nicely sequestered away.

We should test this during CI though when we add more CI to include windows.

Reply:
Perfect, I see now how it is (logically ;P) structured!